executor/aggregate_test.go

```diff
@@ -1237,9 +1237,15 @@ func (s *testSuiteAgg) TestParallelStreamAggGroupConcat(c *C) {
 	tk.MustExec("drop table if exists t;")
 	tk.MustExec("CREATE TABLE t(a bigint, b bigint);")
 
-	for i := 0; i < 10000; i++ {
-		tk.MustExec("insert into t values(?, ?);", rand.Intn(100), rand.Intn(100))
+	var insertSQL string
+	for i := 0; i < 1000; i++ {
```

Comment:
Should we set the init_chunk_size and max_chunk_size to small values since we want to test the parallel scenario in this test?